manim/mobject/types/vectorized_mobject.py

```diff
@@ -611,10 +618,10 @@ def consider_points_equals_2d(self, p0, p1):
         return True
 
     # Information about line
-    def get_cubic_bezier_tuples_from_points(self, points):
+    def get_cubic_bezier_tuples_from_points(self, points: List[Vector]) -> (List[Vector], List[Vector], List[Vector], List[Vector]):
```

Comment:
Function called only demands a Sequence (see below)
```suggestion
    def get_cubic_bezier_tuples_from_points(self, points: typing.Sequence[Vector]) -> (List[Vector], List[Vector], List[Vector], List[Vector]):
```